test/lifecycle/TimeFrame/TimeFrame.test.js

```diff
@@ -0,0 +1,144 @@
+const { time, shouldFail } = require('openzeppelin-test-helpers');
+const { expectBNIsWithinDelta } = require('../../helpers/expectBNIsWithinDelta');
+
+const TimeFrameMock = artifacts.require('TimeFrameMock');
+
+contract('TimeFrame', () => {
+  beforeEach(async () => {
+    this.now = await time.latest();
+
+    this.oneWeek = await time.duration.weeks(1);
+    this.twoWeeksAgo = await this.now - time.duration.weeks(2);
```

Comment:
These are `BN`s, so you'll need to use `.sub` (and `add`) instead of `-`, e.g. `await this.now.sub(time.duration.weeks(2))`